system-variables.md

```diff
@@ -1,129 +1,836 @@
+
+### tidb_wait_split_region_finish
+
+- Scope: SESSION
+- Default value: 1, indicating returning the result after all Regions are scattered.
+- It usually takes a long time to scatter Regions, which is determined by PD scheduling and TiKV loads. This variable is used to set whether to return the result to the client after all Regions are scattered completely when the `SPLIT REGION` statement is being executed. Value `0` indicates returning the value before finishing scattering all Regions.
+- Note that when scattering Regions, the write and read performances for the Region that is being scattered might be affected. In batch-write or data importing scenarios, it is recommended to import data after Regions scattering is finished.
+
+### tidb_wait_split_region_timeout
+
+- Scope: SESSION
+- Default value: 300
+- This variable is used to set the timeout for executing the `SPLIT REGION` statement. The unit is second. If a statement is not executed completely within the specified time value, a timeout error is returned.
+
+### tidb_window_concurrency <span class="version-mark">New in v4.0</span>
+
+- Scope: SESSION | GLOBAL
+- Default value: 4
+- This variable is used to set the concurrency degree of the window operator. 
+
+### time_zone
+
+- Scope: SESSION | GLOBAL
+- Default value: SYSTEM
+- This variable sets the sytem time zone. Values can be specified as either an offset such as '-8:00' or a named zone 'America/Los_Angeles'.
+
+### transaction_isolation
+
+- Scope: SESSION | GLOBAL
+- Default value: REPEATABLE-READ
+- This variable sets the transaction isolation. TiDB advertises `REPEATABLE-READ` for compatibility with MySQL, but the actual isolation level is Snapshot Isolation. See [transaction isolation levels](/transaction-isolation-levels.md) for further details.
+
+### tx_isolation
+
+This variable is an alias for _transaction_isolation_.
+
+### version
+
+- Scope: NONE
+- Default value: 5.7.25-TiDB-(tidb version)
+- This variable returns the MySQL version, followed by the TiDB version. For example '5.7.25-TiDB-v4.0.0-beta.2-716-g25e003253'.
+
+### version_comment
+
+- Scope: NONE
+- Default value: (string)
+- This variable returns additional details about the TiDB version. For example, 'TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible'.
+
+### wait_timeout
+
+- Scope: SESSION | GLOBAL
+- Default value: 0
+- This variable controls the idle timeout of user sessions in seconds. A zero-value means unlimited.
 
-## TiDB specific system variables
+### windowing_use_high_precision
 
-See [TiDB Specific System Variables](/tidb-specific-system-variables.md).
+- Scope: SESSION | GLOBAL
+- Default value: ON
+- This variable controls whether to use the high precision mode when computing the window functions.
```

Comment:
@SunRunAway Could you help us to explain more on the computing difference between `on` and `off`? Would it execute faster when it's set to `off`?